src/wallet/sigmaspendbuilder.cpp

```diff
@@ -105,6 +105,10 @@ static std::unique_ptr<SigmaSpendSigner> CreateSigner(const CSigmaEntry& coin)
 
 SigmaSpendBuilder::SigmaSpendBuilder(CWallet& wallet, const CCoinControl *coinControl) : TxBuilder(wallet)
 {
+    if (!zwalletMain) {
+        throw std::logic_error("Sigma feature required HD wallet");
```

Comment:
should be just `require`

Reply:
I'm not grammar mastery so let's ask other people. @reubenyap what do you think?

Reply:
@riordant just provided a different one `Sigma feature requires HD wallet`.

Reply:
@riordant suggestion is okay